src/views/Visualizer/View/View.js

```diff
@@ -313,6 +313,19 @@ export class Visualizer extends React.Component {
     this.updateLayersTree();
   }
 
+  onMapUpdate = () => {
+    // Add a class to the body for easier automatic tools detection
+    document.body.classList.add('cy-isloaded');
```

Comment:
```suggestion
    document.body.classList.add('tile-isloaded');
```
maybe you should use mre generic class name ?